tools/plink_load_variant_file.cwl

```diff
@@ -0,0 +1,69 @@
+      plink
+inputs:
+  input_vcf: { type: 'File?', inputBinding: { position: 2, prefix: "--vcf" }, doc: "Input VCF file. Can be bgzipped." }
+  input_bcf: { type: 'File?', inputBinding: { position: 2, prefix: "--bcf" }, doc: "Input BCF file. Can be bgzipped." }
+  output_basename: { type: 'string?', default: "plink", inputBinding: { position: 2, prefix: "--out"}, doc: "Basename for plink binary files." }
+  double_id: { type: 'boolean?', inputBinding: { position: 2, prefix: "--double-id" }, doc: "causes both family and within-family IDs to be set to the sample ID" }
+  const_fid: { type: 'string?', inputBinding: { position: 2, prefix: "--const-fid" }, doc: "converts sample IDs to within-family IDs while setting all family IDs to a single value" }
+  id_delim: { type: 'string?', inputBinding: { position: 2, prefix: "--id-delim" }, doc: "causes sample IDs to be parsed as <FID><delimiter><IID>" }
+  vcf_idspace_to: { type: 'string?', inputBinding: { position: 2, prefix: "--vcf-idspace-to" }, doc: "convert all spaces in sample IDs to this character" }
+  biallelic_only: { type: ['null', { type: enum, name: "biallelic_only", symbols: ["base", "strict", "list"] }], inputBinding: { position: 2, prefix: "--biallelic-only", shellQuote: false, valueFrom: "$(self == 'base' ? '' : self)"}, doc: "Use to skip all variants where at least two alternate alleles are present in the dataset. The 'strict' mode will indiscriminately skip variants with 2+ alternate alleles listed even when only one alternate allele actually shows up. The 'list' mode will dump a list of skipped variant IDs to plink.skip.3allele." }
+  vcf_min_qual: { type: 'float?', inputBinding: { position: 2, prefix: "--vcf-min-qual" }, doc: "causes all variants with QUAL value smaller than the given number, or with no QUAL value at all, to be skipped" }
+  vcf_filter: { type: 'string[]?', inputBinding: { position: 2, prefix: "--vcf-filter" }, doc: "skip variants which failed one or more filters tracked by the FILTER field" }
+  vcf_require_gt: { type: 'boolean?', inputBinding: { position: 2, prefix: "--vcf-require-gt" }, doc: "Skip variants with missing GT fields." }
+  vcf_min_gp: { type: 'float?', inputBinding: { position: 2, prefix: "--vcf-min-gp" }, doc: "excludes all genotype calls with GP value below the given threshold" }
+  vcf_min_gq: { type: 'float?', inputBinding: { position: 2, prefix: "--vcf-min-gq" }, doc: "excludes all genotype calls with GQ below the given (nonnegative, decimal values permitted) threshold" }
+  vcf_half_call: { type: ['null', { type: enum, name: "vcf_half_call", symbols: ["error", "haploid", "missing", "reference"]}], inputBinding: { position: 2, prefix: "--vcf-half-call" }, doc: "specify how '0/.' and similar GT values should be interpreted. Error: PLINK 1.9 errors out and reports the line number of the anomaly. Haploid: Treat half-calls as haploid/homozygous. Missing: Treat half-calls as missing. Reference: Treat the missing part as reference." }
+
+  cpu:
```

Comment:
Why switch styles here, from inline to multi?

Reply:
No reason in particular. Can squash.